opslib/osm/utils.py

```diff
@@ -0,0 +1,18 @@
+__all__ = ["hash_from_dict"]
+
+import hashlib
+import json
+from typing import Any, Dict, List
+
+
+def hash_from_dict(dict: Dict[str, Any]) -> str:
+    """Get a hash from a dictionary"""
+    dict_str = json.dumps(dict, sort_keys=True)
+    result = hashlib.md5(dict_str.encode())
+    return result.hexdigest()
+
+
+def find_in_list_with_key(list: List[Dict[str, Any]], key: str, value: str) -> Any:
+    for item in list:
+        if key in item and item[key] == value:
+            return item
```

Comment:
Default return should be added for easier understanding that if the item is not found a None should be returned

Reply:
done. Good idea btw